### PR TITLE
chore: update to readme

### PR DESCRIPTION
**Pull request by jpvajda**
this PR updates the readme and fixing some MD linter errors. 
this PR also seems to have an update to newrelic.go file adding some print statements, though I didn't make those locally

### CODE REVIEW

File: newrelic/newrelic.go [hunk 1]
```diff
@@ -1,3 +1,8 @@
+fmt.Print("\n\n **************************** \n")
```
Comment (zlesnr) [inline newrelic/newrelic.go:1]: Probably remove these once you're done testing.

Comment (jpvajda) [reply newrelic/newrelic.go:1]: Interesting, I didn't add those lines, perhaps they came in from another unmerged PR?

Comment (jpvajda) [reply newrelic/newrelic.go]: fixed @zlesnr

**Review by ctrombley — DISMISSED** [review]
:shipit:

File: README.md [hunk 2]
```diff
@@ -164,21 +162,19 @@ This refers to what part of the code is the focus of the work.  For example:
...
-```
+``` bash
 $ make docs
 ```
 
-
 ## Community Support
 
-New Relic hosts and moderates an online forum where you can interact with New Relic employees as well as other customers to get help and share best practices. Like all New Relic open source community projects, there's a related topic in the New Relic Explorers Hub. You can find this project's topic/threads here:
+New Relic hosts and moderates an online forum where you can interact with New Relic employees as well as other customers to get help and share best practices. Like all New Relic open source community projects, there's a related topic in the New Relic Explorers Hub. You can find our Team's project topic/threads here:
```
Comment (sanderblue) [inline README.md:175]: Minor suggestion

```diff
- Team's
+ team's
```

File: README.md [hunk 3]
```diff
@@ -164,21 +162,19 @@ This refers to what part of the code is the focus of the work.  For example:
...
 
-[https://discuss.newrelic.com/t/new-relic-one-top-nerdpack/82934](https://discuss.newrelic.com/t/new-relic-one-top-nerdpack/82934)
+[Developer Tool Kit](https://discuss.newrelic.com/t/about-the-developer-toolkit-category/90159)
```
Comment (sanderblue) [inline README.md:177]: I think we've been using `Toolkit` as one word.

Comment (jpvajda) [inline README.md:177]:
```suggestion
[Developer ToolKit](https://discuss.newrelic.com/t/about-the-developer-toolkit-category/90159)
```

File: README.md (hunk 2 above)
Comment (jpvajda) [inline README.md:175]:
```suggestion
New Relic hosts and moderates an online forum where you can interact with New Relic employees as well as other customers to get help and share best practices. Like all New Relic open source community projects, there's a related topic in the New Relic Explorers Hub. You can find our team's project topic/threads here:
```

**Review by sanderblue — APPROVED** [review]

**Pull request merged**